packages/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -42,59 +45,175 @@ import 'package:amplify_flutter/amplify.dart';
 
-/// {@template authenticator.authenticator}
+/// {@template amplify_authenticator.authenticator}
 /// # Amplify Authenticator
 ///
-/// A widget that allows customers to authenticate their apps.
+/// A prebuilt sign in/sign up experience for Amplify Auth. Simply provide a
+/// [child] widget which is your app's authenticated route, and the Authenticator
+/// will handle the logic behind managing user sessions and guiding users through
+/// the sign up and sign in process.
+///
+/// ```dart
+/// import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
+/// import 'package:amplify_authenticator/amplify_authenticator.dart';
+/// import 'package:amplify_flutter/amplify.dart';
+/// import 'package:flutter/material.dart';
+///
+/// import 'amplifyconfiguration.dart';
+///
+/// void main() {
+///   runApp(const MyApp());
+/// }
+///
+/// class MyApp extends StatefulWidget {
+///   const MyApp({Key? key}) : super(key: key);
+///
+///   @override
+///   _MyAppState createState() => _MyAppState();
+/// }
+///
+/// class _MyAppState extends State<MyApp> {
+///   @override
+///   void initState() {
+///     super.initState();
+///     _configureAmplify();
+///   }
+///
+///   Future<void> _configureAmplify() async {
+///     try {
+///       await Amplify.addPlugin(AmplifyAuthCognito());
+///       await Amplify.configure(amplifyconfig);
+///     } on Exception catch (e) {
+///       print('Could not configure Amplify: $e');
+///     }
+///   }
+///
+///   @override
+///   Widget build(BuildContext context) {
+///     return MaterialApp(
+///       theme: ThemeData.light(),
+///       darkTheme: ThemeData.dark(),
+///       home: Authenticator(
+///         child: const LoggedInScreen(),
+///       ),
+///     );
+///   }
+/// }
+///
+/// class LoggedInScreen extends StatelessWidget {
+///   const LoggedInScreen({Key? key}) : super(key: key);
+///
+///   @override
+///   Widget build(BuildContext context) {
+///     return Scaffold(
+///       body: Center(
+///         child: Column(
+///           children: const [
+///             Text('Logged In'),
+///             SignOutButton(),
+///           ],
+///         ),
+///       ),
+///     );
+///   }
+/// }
+/// ```
+/// {@endtemplate}
+///
+/// ## Customization
+///
+/// ### Themeing
 ///
-/// The Authenticator widget requires one argument, a child widget,
-/// to define the initial authentication flow.
+/// By default, the Authenticator uses your app's Material theme for its styling.
+/// However, you can also use the a hand-crafted Amplify theme by setting
+/// [useAmplifyTheme] to `true`.
 ///
-/// This authenticator accepts the following custom forms, sign in, sign up and confirm sign in.
+/// ### Forms
 ///
-/// Note that working with custom forms is optional. Thus, if no additional arguments
-/// are passed, the authenticator will defined the following default forms with their
-/// respective form fields:
+/// The Authenticator uses your app's Cognito configuration to determine which
```

Comment:
I would say something like 'your app's Cognito configuration as defined in the amplifyconfiguration.dart file', or even just 'your app's amplifyconfiguration.dart file.' We want to make this clear for customers who are using existing resources and writing their own config.

Reply:
Fixed - thanks! Good call.